asp_interactive_configuration/ooasp/config.py

```diff
@@ -431,7 +445,24 @@ def format_cv(self,cv:Predicate):
         """
         args = [str(a) for a in cv.args.symbol.arguments]
         return f"Object {cv.object_id}: {cv.info.format(*args)}"
+    
+    def all_as_user(self):
+        """
+        All configuration editable facts as user symbols
 
+        Returns:
+            list[Symbol]: User symbols
+        """
+        return [Function("user",[f.symbol]) for f in  self.editable_facts]
+    
+    def consider_as_user(self, facts):
+        """
+        All configuration editable facts are added as user symbols
```

Comment:
I don't understand this docstring. Shouldn't it be "The given facts" rather than "All configuration editable facts"?